src/python/grapl-template-generator/grapl_template_generator/grapl-templates/rust-grpc-service/{{cookiecutter.project_slug}}/.cargo/config.toml

```diff
@@ -0,0 +1,7 @@
+[build]
+rustflags = [
+    "--deny=warnings",
+]
+
+[http]
+ssl-version.min = "tlsv1.2"
```

Comment:
I'd rather not introduce a bunch of customized configuration for each Rust sub-project.